templates/base.html

```diff
@@ -26,7 +27,9 @@
 
   {% bootstrap_css %}
 
+  {% compress css %}
```

Comment:
Getting error from template rendering ([traceback](http://dpaste.com/3AD7YYW))

Reply:
I think you don't have Redis setup. According to the .env file, `REDIS_URL=redis://localhost:6379`

Reply:
My bad, Forgot to start redis server.

Reply:
No probs buddy :)